Enterprise-CMCS/macpro-onemac
# OY2 25098: Move Administrative Package Changes Section

Story: https://qmacbis.atlassian.net/browse/OY2-25098
Endpoint: See github-actions bot comment

### Details
The Administrative Package Changes Section should be at the very bottom of the package details page.

### Changes
- moved the administrative changes section to bottom of details page
- again, removed extraneous console.logs
- removed commented out tests
- updated a test to look at a different package

### Test Plan
1. Login to OneMAC as any user that can see packges
2. Open the package details of a package with an Administrative Change (note, RAI Response Enabling is done through an Administrative Change)
3. Verify that the Administrative Change section is at the bottom of the details page.
